packages/airnode-node/src/workers/local-gateways/validation.ts

```diff
@@ -146,7 +146,7 @@ export const signOevDataBodySchema = z.object({
   updateId: z.string(),
   bidderAddress: z.string(),
   bidAmount: z.string(),
-  signedData: z.array(beaconSchema),
+  beacons: z.array(beaconSchema),
```

Comment:
Another unnecessary cosmetic change that wouldn't be needed if you wouldn't introduce the first one.